src/ethereum/mod.rs

```diff
@@ -382,21 +406,36 @@ impl Ethereum {
         TX_COUNT.with_label_values(&[&bytes4]).inc();
 
         // Send TX to mempool
-        let pending = self
-            .provider
-            .send_transaction(tx.clone(), None)
-            .await
-            .map_err(|error| {
-                error!(?nonce, ?error, "Failed to send transaction");
-                TxError::Send(Box::new(error))
-            })?;
+        let span = span!(Level::INFO, "Send TX to mempool");
```

Comment:
Does it create spans with whitespace, if so maybe we could avoid that ?

Reply:
This ends up as the log message (so we need it to be somewhat human readable for the log to make sense). In datadog it looks something like this https://app.datadoghq.com/apm/trace/3176724204366318672?colorBy=service&env=stage&graphType=flamegraph&shouldShowLegend=true&sort=time&spanID=15278035570635956810&spanViewType=metadata&timeHint=1658904095107.319

Reply:
What I don't like is the `opentelemetry.internal` noise everywhere. Maybe we can replace this with the `target` field or something more useful.